### ISSUE TITLE
Disabling channels/instruments

### ISSUE THREAD

**Issue by Ep8Script** [deskjet/chiptune2.js#41]
It would be cool if it were possible to disable specific channels or instruments with this library (or libopenmpt), but it doesn't seem to be possible. I've tried a lot of things (including reading up on some information [here](https://forum.openmpt.org/index.php?topic=6277.0), but I've unfortunately been unable to get it to work.

Is there something I'm missing, or is this not possible? Thanks!

**Comment by deskjet** [comment]
I'm sure it can be done. libopenmpt can supports muting channels using the [`set_channel_mute_status`](https://lib.openmpt.org/doc/structopenmpt__module__ext__interface__interactive.html#a874f7ea95fdb65c470fac5259c1ddb75) function. However, I can't figure out how to call `openmpt_module_ext_get_interface` from JS correctly to obtain the `mod_ext` pointer. I'll experiment later today if there is time.

**Comment by Ep8Script** [comment]
Yeah that's where I got stuck unfortunately. Appreciate you taking the time anyway, good luck! Thanks for making this project.

**Comment by christiancodes** [comment]
I'm investigating this too. Doesn't look like `set_channel_mute_status` has made it into the libopenmpt.js conversion.

**Comment by deskjet** [comment]
I did a bit of research and I think we might have to create some bindings so we can access the `openmpt_module_ext_interface_interactive` struct from JS and call the functions pointers there using Emscripten's `dynCall`. Not sure whether it'll work though.

**Comment by ramdanirfox** [comment]
@deskjet i prefer to use wasmTable.get(functionPointer) because i can apply argument more flexible.
